### DISCUSSION TITLE
BT-717  Clean Vehicles Directive does not show in the publication

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#822] [Q&A]
BT-717  Clean Vehicles Directive does not show in the publication (because it is dependent on conditions that are  related to BT-717. I think that 'BT-717 is present' should be added tot the conditions of point 7 (the first line in the screenshot). And if so, i think it should be changed in a number of view templates

![image](https://github.com/OP-TED/eForms-SDK/assets/111063502/422fcd12-9890-43c0-bc71-d3dad5edd318)

Regards Peter van de Kletersteeg, TenderNed, The Netherlands

**Comment by pdonohoe** [comment]
Thank you very much Peter for spotting this bug, and for providing the solution. It will be fixed in the next release of the SDK, 1.11
